tests/test_elx/test_catalog.py

```diff
@@ -103,3 +152,50 @@ def test_catalog_deselect_property(tap: Tap):
     catalog_dict = catalog.dict(by_alias=True)
 
     assert catalog_dict["streams"][0]["metadata"][0]["metadata"]["selected"] == False
+
+
+def test_catalog_replication_method(tap_incremental: Tap):
+    """If we have an incremental stream, the replication_method in the catalog should be `INCREMENTAL`."""
+    catalog_dict = tap_incremental.catalog.dict(by_alias=True)
+
+    replication_method = INCREMENTAL_CATALOG["streams"][0]["replication_method"]
+    assert replication_method == "INCREMENTAL"
+    assert catalog_dict["streams"][0]["replication_method"] == "INCREMENTAL"
```

Comment:
Ik zou hier de waarde uit beide met elkaar vergelijken, ipv kijken of ze beide incremental zijn.